INPUT_DATA/re-format_data.py

```
@@ -0,0 +1,225 @@
+        "INSERT INTO STUDENT_INFO (UserID, Form, House, TimeTableID) VALUES (?,?,?,?)",
+        (pupil['Pupil ID'], pupil['Form'], pupil['Boarding House'], TTID)
+        )
+
+data = DB_interface.get_data("SELECT UserID FROM SUBJECTS")
+
+data = [row[0] for row in data]
+
+for teach in set(data):
+    print(teach)
+    f = "FirstName"+str(teach)
+    l = "LastName"+str(teach)
```

Ruff (0.14.7)

Ambiguous variable name: `l`

(E741)